src/ucp/rma/rma_sw.c

```diff
@@ -77,11 +85,19 @@ static ucs_status_t ucp_rma_sw_progress_get(uct_pending_req_t *self)
     ucs_status_t status;
     ssize_t packed_len;
 
+    /* make an asuumption here that EP was able to send the GET REQ, since there
+     * are transports (e.g. SELF - it does send-recv in the AM function) that is
+     * able to complete the remote request operation inside uct_ep_am_bcopy()
+     * and decrement the flush_ops_count before it was incremented */
```

Comment:
can we avoid copy/paste of this comment for put,get,rma?

Reply:
done